scripts/test/generate_lld_tests.py

```diff
@@ -72,8 +72,8 @@ def generate_wast_files(clang_bin, lld_bin, emscripten_root):
 
 
 if __name__ == '__main__':
-  if len(sys.argv) != 4:
-    print 'Usage: generate_lld_tests.py [path/to/clang] [path/to/lld] \
-[path/to/emscripten]'
+  print(shared.options.positional_args)
```

Comment:
Do we want to print positional args always, or is this a holdover from debugging?